server/repository/src/db_diesel/backend_plugin_row.rs

```diff
@@ -8,10 +8,10 @@ use diesel_derive_enum::DbEnum;
 use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
-#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+#[serde(rename_all = "snake_case")]
```

Comment:
snake case now because this maps directly to main plugins type now, made comments in types/mod.rs